README.md

```diff
@@ -66,13 +66,13 @@ This project is released under the [Apache 2.0 license](LICENSE).
 - Add abstract and sketch of the papers in readmes
 - Fix some weight initialization bugs
 
-Please refer to [changelog.md](docs/changelog.md) for details and release history.
+Please refer to [changelog.md](docs/en/changelog.md) for details and release history.
```

Comment:
Please pull the master and remove these diff fiels

Reply:
Master is 1 commit ahead dev-v2.20.0